capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst

```diff
@@ -232,6 +232,15 @@ Breaking Changes
 
 QR code scanning UI and functionality are changed. Scanning and processing happens automatically. You can find more details `here <features.html#qr-code-scanning>`_.
 
+Review
+~~~~~
+
+Breaking Changes
+++++++++++++++++
+
+Review screen doesn't support re-ordering the images anymore.
+If processing of images fails user is redirected to Error screen.
+
```

Comment:
[Request] We also need a section for new features:
* custom loading indicator on the "process" button and how to customise it
* bottom navigation bar and how to customise it

For both it's enough to add a short sentence on how to set the custom implementation using `GiniCapture`. Like we did for "Help" or "Analysis".